apps/condo/domains/organization/constants/common.js

```diff
@@ -247,6 +254,15 @@ const DEFAULT_ROLES = {
         'canReadSettings': true,
         'canReadExternalReports': true,
         'canReadServices': true,
+        'canReadPaymentsWithInvoices': false,
+        'canReadMarketplace': false,
+        'canManageMarketplace': false,
+        'canReadInvoices': true,
+        'canManageInvoices': false,
+        'canReadMarketItemPrices': true,
+        'canReadMarketPriceScopes': true,
+        'canReadMarketItems': true,
+        'canManageMarketItems': false,
```

Comment:
Foreman can read Market Items and cannot Read Marketplace? Is everything correct here?

Reply:
fixed it